base/error.go

```diff
@@ -38,6 +39,22 @@ func ErrorAsHTTPStatus(err error) (int, string) {
 	if err == nil {
 		return 200, "OK"
 	}
+
+	switch err.Error() {
+	case gocbcore.ErrKeyNotFound.Error():
+		return http.StatusNotFound, "missing"
+	case gocbcore.ErrKeyExists.Error():
+		return http.StatusConflict, "Conflict"
+	case gocbcore.ErrTimeout.Error():
+		return http.StatusServiceUnavailable, "Database server is over capacity"
```

Comment:
Is 'over capacity' actually the appropriate message for a timeout error?  It might be the root cause, but maybe not in all cases?